### PR TITLE
Fixing Tubi & Stirr

### LINKED ISSUE #14188: Broken: FOX 7 Austin TX (KTBC)
### Broken Link

https://csm-e-eb.csm.tubi.video/csm/extlive/tubiprd01,p-KTBC-Monetizer.m3u8

### What happened to the stream?

Not loading

### Possible Replacement (optional)

_No response_

### Notes (optional)

Here is the stream on tubi https://tubitv.com/live/555112/fox-7-austin also the official website stream https://www.fox7austin.com/live

### Contributing Guide

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md)

### PR DESCRIPTION
**Pull request by brokenstream**
closes #14188

### CODE REVIEW

File: streams/us_tubi.m3u [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #EXTM3U
 #EXTINF:-1 tvg-id="EstrellaNews.us",Estrella News (720p)
-https://csm-e-eb.csm.tubi.video/csm/live/247083838.m3u8
+https://csm-e-eb.csm.tubi.video/csm/extlive/tubiprd01,Cloudfront-Estrella.m3u8
```
Comment (BellezaEmporium) [inline streams/us_tubi.m3u:3]: link redirects to TS playlist files, however master https://lnc-estrella.tubi.video/playlist.m3u8 works.

File: streams/us_tubi.m3u [hunk 2]
```diff
@@ -10,53 +10,49 @@ https://live-news-manifest.tubi.video/live-news-manifest/csm/extlive/tubiprd01,F
 #EXTINF:-1 tvg-id="KCCIDT1.us",KCCI-TV News Des Moines IA (720p)
 https://live-news-manifest.tubi.video/live-news-manifest/csm/extlive/tubiprd01,Cloudfront-KCCI.m3u8
 #EXTINF:-1 tvg-id="KCPQDT1.us",FOX Q13 Seattle WA (KCPQ) (720p)
-https://csm-e-eb.csm.tubi.video/csm/extlive/tubiprd01,p-KCPQ-Monetizer.m3u8
+https://live-news-manifest.tubi.video/live-news-manifest/csm/extlive/tubiprd01,KCPQ.m3u8
```
Comment (BellezaEmporium) [inline streams/us_tubi.m3u:13]: link redirects to TS playlist files, however master https://lnc-kcpq.tubi.video/index.m3u8 works.

File: streams/us_tubi.m3u [hunk 3]
```diff
@@ -1,109 +1,107 @@
...
 #EXTINF:-1 tvg-id="WBALDT1.us",WBAL-TV News Baltimore MD (576p)
-https://live-news-manifest.tubi.video/live-news-manifest/csm/extlive/tubiprd01,Cloudfront-WBAL.m3u8
+https://lnc-wbal.tubi.video/playlist.m3u8
 #EXTINF:-1 tvg-id="WCVBDT1.us",WCVB-TV News Boston MA (720p) [Not 24/7]
-https://live-news-manifest.tubi.video/live-news-manifest/csm/extlive/tubiprd01,Cloudfront-WCVB.m3u8
+https://lnc-wcvb.tubi.video/playlist.m3u8
 #EXTINF:-1 tvg-id="WDSUDT1.us",NBC 6 New Orleans LA (WDSU) (720p)
-https://csm-e-eb.csm.tubi.video/csm/extlive/tubiprd01,Cloudfront-WDSU.m3u8
+https://lnc-wdsu.tubi.video/playlist.m3u8
 #EXTINF:-1 tvg-id="WeatherNation.us",Weathernation (720p) [Geo-blocked]
```
Comment (BellezaEmporium) [inline streams/us_tubi.m3u:64]:
```suggestion
#EXTINF:-1 tvg-id="WeatherNation.us",Weathernation (720p)
```
Works from a French IP.

File: streams/us_tubi.m3u [hunk 4]
```diff
@@ -1,109 +1,107 @@
...
-https://live-news-manifest.tubi.video/live-news-manifest/csm/extlive/tubiprd01,KTVU.m3u8
+https://lnc-ktvu.tubi.video/index.m3u8
 #EXTINF:-1 tvg-id="LiveNOWfromFOX.us",LiveNOW from FOX (720p)
 https://lnc-fox-live-now.tubi.video/index.m3u8
 #EXTINF:-1 tvg-id="Localish.us",Localish (720p)
-https://live-news-manifest.tubi.video/live-news-manifest/csm/extlive/tubiprd01,Localish.m3u8
+https://lnc-localish.tubi.video/index.m3u8
 #EXTINF:-1 tvg-id="News12NewYork.us",News 12 New York (1080p)
 https://lnc-news12.tubi.video/index.m3u8
 #EXTINF:-1 tvg-id="RealMadridTV.es",Real Madrid TV US Version (720p) [Not 24/7]
```
Comment (BellezaEmporium) [inline streams/us_tubi.m3u:46]:
```suggestion
#EXTINF:-1 tvg-id="RealMadridTV.es",Real Madrid TV US Version (720p) [Not 24/7] [Geo-blocked]
```
No for me (France)

File: streams/us_tubi.m3u [hunk 5]
```diff
@@ -1,109 +1,107 @@
...
-https://live-news-manifest.tubi.video/live-news-manifest/csm/extlive/tubiprd01,Real-Madrid.m3u8
+https://lnc-real-madrid.tubi.video/playlist.m3u8
 #EXTINF:-1 tvg-id="Reelz.us",Reelz
-https://live-news-manifest.tubi.video/live-news-manifest/csm/extlive/tubiprd01,Reelz.m3u8
+https://lnc-reelz.tubi.video/playlist.m3u8
 #EXTINF:-1 tvg-id="USATODAY.us",USA Today (720p)
 https://live-news-manifest.tubi.video/live-news-manifest/csm/extlive/tubiprd01,Cloudfront-USA-Today.m3u8
```
Comment (BellezaEmporium) [inline streams/us_tubi.m3u:51]:
```suggestion
https://lnc-usa-today.tubi.video/playlist.m3u8
```

**Review by BellezaEmporium — COMMENTED** [review]
Thank you for this massive refix of channels. Please find comments attached above.

**Review by BellezaEmporium — APPROVED** [review]

**Review by Alstruit — APPROVED** [review]

**Pull request merged**